scss/_base_forms.scss

```diff
@@ -236,9 +236,11 @@ $input-margin-bottom: $spv-inter--scaleable - $spv-nudge * 2;
   // Fieldset styles
   fieldset {
     background-color: $color-light;
-    border: 0;
+    border: 1px solid $color-mid-light;
     border-radius: $border-radius;
     color: $color-dark;
+    padding: $spv-intra $sph-intra--condensed;
+    margin-bottom: $spv-intra * 3;
```

Comment:
This should be `$spv-inter--scaleable` - it's 1.5rem by default but can be adjusted with `$multi`